pyqt_sql_demo/connection/model.py

```diff
@@ -129,6 +173,19 @@ def fetch_more(self):
         # And enable otherwise
         self.fetch_changed.emit(len(more) >= limit)
 
+    def _rows_to_string(self, rows):
+        """ bin data to string """
+        rows_string = []
+        for row in rows:
+            row_string = []
+            for cell in row:
+                if isinstance(cell, typing.ByteString):
```

Comment:
It is also possible to use `bytes` here instead of generic `typing.ByteString`. Since `typing` module is imported for this check only it would make sense to replace the argument with `bytes`.